plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/ViewMaterializationCache.java

```diff
@@ -68,8 +68,8 @@ public TableInfo getCachedTable(BigQueryClient client, String query, Duration vi
 
     private TableId createDestinationTable(TableId remoteTableId)
     {
-        String project = viewMaterializationProject.orElse(remoteTableId.getProject());
-        String dataset = viewMaterializationDataset.orElse(remoteTableId.getDataset());
+        String project = viewMaterializationProject.orElseGet(remoteTableId::getProject);
+        String dataset = viewMaterializationDataset.orElseGet(remoteTableId::getDataset);
```

Comment:
(X) getter

Reply:
as above (getter to a library method, so it's a cheap getter today, may not be a cheap getter tomorrow)